js/__tests__/CameraRollTest.js

```diff
@@ -34,4 +34,9 @@ describe('CameraRoll', () => {
     await CameraRoll.getPhotos({first: 0});
     expect(NativeModule.getPhotos.mock.calls).toMatchSnapshot();
   });
+
+  it('Should call getPhotosFast', async () => {
```

Comment:
Might be a bit more accurately named "should call NativeModules.getPhotos when calling getPhotosFast"

Reply:
Renamed to `Should call native modules getPhotos when calling getPhotosFast`